config/repo-values.yaml

```diff
@@ -366,3 +366,5 @@ sync:
       url: https://dacruz21.github.io/helm-charts
     - name: kuberhealthy
       url: https://comcast.github.io/kuberhealthy/helm-repos
+    - name: neo4j-community
+      url: https://equinor.github.io/neo4j-charts/charts/
```

Comment:
Since this is for the company equinor did you want to name the repo by that name? That way it represents the company and not the project.

Reply:
It's a huge company with hundreds of teams on our GitHub so initially I did not want to be the one in charge of "the one" helm repo. I cannot guarantee for example that everyone will remember to follow best practices and linting. But on second thought I might do that and create some instructions on how it should be used if you think that would be acceptable?

Reply:
I can spend a bit more time and set up GH Actions to force linting on PRs to our helm repo for example.

Reply:
@StianOvrevage That's fine. I just didn't know if you wanted your company name reflected in the repo name that's listed rather than the project. We usually expect the repo name to reflect the organization sharing.

Where I'm at it's Samsung which is huge. So, we use our team name as well. We use something like `samsung-cnct`. On the team we can share and enforce our rules.

Reply:
I went ahead and created a common repo for our helm charts. Set up GH actions to lint and build packages and helm indexes.

I've updated the PR to reflect the new repo and name.